module-1/pandas-project/your-code/Pandas project.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
```

Comment:
1. It's a good practice once you have imported the dataset to your working notebook , to have a print of dataframe info and main statistics through describe method. 
2. Please in next notebook exercises make the pull request with the executed notebook. 
3. In[3] : Really good practice to use accesors like loc to make the code more readable. 
4. In[5]: Good approach, depending the kind of analysis you are running, it could be useful to have a year variable in case of temporal data types. 
5. In[8]: Why did you create an user-defined function for converting to datetime type? Did you face any issue doing it directly? 
6. In[6]: When you slice a dataframe with a high number of columns it's better to use accesors instead of typing every single column label. 
7. In real world, dropping null values should be analysed in a deeper way. Dealing with missing values is on of the most tricky things in data cleaning. Dropping or filling them with zeros can cause unexpected and not desired results in further steps of the data science process. The drop reason should be based on a very robust hypothesis.
8. Really good approach trying to fill NA values with the media and/or the median. Sometimes is useful to analyse outliers and run the bining for numeric values without missing values. Depending on the results it could be 
9. The last two questions are a little bit confusing. If you had created a year column in the very begining you could have simplify this last part of the analysis.